llvm_ir_dataset_utils/builders/portage_builder.py

```diff
@@ -46,6 +46,27 @@ def generate_emerge_command(package_to_build, threads, build_dir):
   return command_vector
 
 
+def perform_build_again(package_name, assembled_build_command, corpus_dir,
```

Comment:
Why do we need a separate function to perform the build again? Why can't we just have build call itself recursively? 

Also, why does this exist in the first place? Are you seeing build non-determinism? That should be fixed in upstream Gentoo (assuming feasible) rather than here.